### DISCUSSION TITLE
VSCode Remote SSH plugin vs. Dreamhost shared hosting

### DISCUSSION THREAD

**Discussion by giscus** [davidsneighbour/kollitsch.dev#220] [Comments]
# VSCode Remote SSH plugin vs. Dreamhost shared hosting

This is the online playground of Patrick Kollitsch, David's Neighbour, in Germany produced, digital native living & working on Ko Samui, Thailand.

https://kollitsch.dev/blog/2022/vscode-remote-ssh-plugin-vs-dreamhost-shared-hosting/

**Comment by jeffaxup** [comment]
Thanks, I had the same problem with connecting to DH, and this solved it. The error messages didn't really point to this solution, so I don't know how you found it. Thanks for posting.

**Comment by alanhoyle** [comment]
Thanks! 

I was seeing the same thing.   This helps tremendously and makes updating my site much easier.

**Comment by evandstark** [comment]
Huge thank you! Love when I find a solution to a strange problem quickly and it works.

**Comment by fcorbett** [comment]
Thank you! Still does the trick (though I used it with Cursor.)
